src/main/java/com/uwetrottmann/trakt5/TraktV2.java

```diff
@@ -44,14 +45,14 @@
      * trakt API v2 URL.
      */
     public static final String API_HOST = "api.trakt.tv";
+    public static final String API_STAGING_HOST = "api-staging.trakt.tv";
     public static final String API_URL = "https://" + API_HOST + "/";
+    public static final String API_STAGING_URL = "https://" + API_STAGING_HOST + "/";
     public static final String API_VERSION = "2";
 
     public static final String SITE_URL = "https://trakt.tv";
     public static final String OAUTH2_AUTHORIZATION_URL = SITE_URL + "/oauth/authorize";
-    public static final String OAUTH2_DEVICE_CODE_URL = API_URL + "oauth/device/code";
-    public static final String OAUTH2_DEVICE_TOKEN_URL = API_URL + "oauth/device/token";
-    public static final String OAUTH2_TOKEN_URL = SITE_URL + "/oauth/token";
+    public static final String OAUTH2_TOKEN_URL = "oauth/token";
```

Comment:
That looks wrong, why was this changed?

Reply:
Relative paths are used in the rest of retrofit interfaces. That paths were exception. I tried to make it all in the same style.

Reply:
Thanks for spotting this (already fixed separately, but forgot to say thanks here), it was wrong this whole time!